packages/@o3r/design/builders/generate-style/helpers/style-renderer-options.ts

```diff
@@ -122,8 +124,23 @@ export const getStyleRendererOptions = (tokenVariableNameRenderer: TokenKeyRende
     }
   })(options.variableType || options.language);
 
+  /** Variable sorting strategy  based on selected language */
```

Comment:
```suggestion
  /** Sorting strategy of variables based on selected language */
```

Reply:
I will take your (mainly for the double space removal :D)